controllers/new.go

```diff
@@ -594,8 +594,10 @@ func (k *kubeResources) newPolicyEngineContainer(instance *cv1.UpdateService, im
 			PeriodSeconds:       10,
 			TimeoutSeconds:      3,
 			Handler: corev1.Handler{
-				TCPSocket: &corev1.TCPSocketAction{
-					Port: intstr.FromInt(8081),
```

Comment:
I was looking into the backstory here, and we've been using TCP sockets [to avoid issues with missing query parameters][1].  That would still be a problem today:

```
$ curl -i https://api.openshift.com/api/upgrades_info/graph
HTTP/2 400 
content-length: 80
content-type: application/json
date: Wed, 05 Jan 2022 18:49:07 GMT
set-cookie: 609f91be912da47026eb733a4b0de2df=a9edca1136b8c149892e9ea5dae71a33; path=/; HttpOnly; Secure; SameSite=None
x-envoy-upstream-service-time: 18
server: envoy

{"kind":"missing_params","value":"mandatory client parameters missing: channel"}
```

But since then, openshift/cincinnati#181 made `version` optional.  And it looks like `arch` and `id` have always been optional, and Cincinnati doesn't care if we pick a non-existent channel:

```console
$ curl 'https://api.openshift.com/api/upgrades_info/graph?channel=does-not-exist'
{"version":1,"nodes":[],"edges":[],"conditionalEdges":[]}
```

So it seems like we could use:

```go
HTTPGet: &corev1.HTTPGetAction{
  Path:   "api/upgrades_info/graph?channel=does-not-exist",
  Port:   intstr.FromInt(8081),
  Scheme: corev1.URISchemeHTTP,
  // optional HTTPHeaders setting Accept: application/vnd.redhat.cincinnati.v1+json
}
```

if we wanted to avoid `/metrics` and stay closer to the client pathways serving actual graphs.

[1]: https://github.com/openshift/cincinnati/pull/83/commits/db9ad4a1c7434fbc82088eb7efd61c40953e1ff3